content/docs/sentry-token/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sentry token
+lastUpdated: '2021-02-23T21:00:00.0Z'
+description: How to create a Sentry Token with permissions for use in Backstage.
+---
+
+## Introduction
+
+In order to use the Backstage Sentry plugin with Roadie, you must securely provide Roadie with an API key which it can use to access the Sentry API.
+
+## Steps
+
+1. Create an internal application in the Sentry UI. Do this at the organization level, rather than the personal level.
```

Comment:
feels like the last sentence needs 'on' word somewhere in between

Reply:
If I remember correctly, I copied this out of the old Google doc we had. I will revamp the whole page now because it's not detailed enough anyway.

Reply:
This is updated now. Here are the new docs: https://deploy-preview-115--roadie.netlify.app/docs/integrations/sentry/